fs/rc/rcserver/rcserver.go

```diff
@@ -384,7 +384,7 @@ func (s *Server) serveRemote(w http.ResponseWriter, r *http.Request, path string
 }
 
 // Match URLS of the form [fs]/remote
-var fsMatch = regexp.MustCompile(`^\[(.*?)\](.*)$`)
+var fsMatch = regexp.MustCompile(`^\[(.*)](\/.*)`)
```

Comment:
I don't think this regexp is correct - it breaks the tests.

Can you explain what you were trying to do and we can see if we can fix it up.

Reply:
One test that fails is using an url such as `[remote:path]file.txt`, i.e. no initial slash. This new regex assumes there is always an initial slash and that this therefore can be used to split `[remote:path]/path/to/object` while accepting `]` in path.